assets/js/view.js

```diff
@@ -0,0 +1,154 @@
+export default class View {
+    constructor() {
+        this.tileDimension;
```

Comment:
Jak nie inicjalizujesz tych zmiennych w konstruktorze, to możesz je podać poza konstruktorem:
```
class MyClass {
  tileDimension;
  innerBorderWidth;
}
```